content/en/docs/distributions/gke/pipelines/enable-gpu-and-tpu.md

```diff
@@ -85,4 +83,6 @@ container:
       tf-version.cloud-tpus.google.com: "1.12"
 ```
 
+See [preemptible_tpu_gpu.py](https://github.com/kubeflow/pipelines/blob/master/samples/core/preemptible_tpu_gpu/preemptible_tpu_gpu.py) for a complete example to run on preemptible TPU or GPU.
```

Comment:
Suggested revision - When creating a link in docs, the link text must be descriptive:

To learn more, see an [example pipeline that uses a preemptible node pool with TPU or GPU](https://github.com/kubeflow/pipelines/blob/master/samples/core/preemptible_tpu_gpu/preemptible_tpu_gpu.py).